README.rst

```diff
@@ -7,15 +7,14 @@ support writes too) efficiently without serialization or deserialization.
 The main supported implementation of obtaining shared memory is by memory-mapping files, but the library also supports
 mapping buffers (anonymous mmap objects) as well, albeit they're harder to share among processes.
 
-Currently, most primitive types and collections are supported, except `dicts`.
-
 Supported primivite types:
 
     * int (up to 64 bit precision)
     * str (bytes)
     * unicode
     * frozenset
     * tuple / list
+    * dict
 
 Primitive types are cloned into their actual builtin objects when accessed. Although fast, it does imply that contianers
 will take up a lot of process-local memory when accessed. Support for collection proxies that take the place of
```

Comment:
Now we support proxied types!.
I would add that we support date, datetime, buffer and numpy arrays too.